Scenarios/TripPin/src/webapi/ODataSamples.WebApiService/Controllers/PeopleController.cs

```diff
@@ -27,6 +27,7 @@ public IHttpActionResult Get()
         }
 
         // GET odata/People('key')
+        [EnableQuery]
```

Comment:
It's unnecessary because the Get() action has the related argument named "queryOptions".

It should use the "queryOptions" in the Get() body to apply the query options for the result.